mypy.ini

```diff
@@ -0,0 +1,50 @@
+# Global options:
+
+[mypy]
+warn_unused_configs = True
+follow_imports = skip
```

Comment:
What are your thoughts between `skip` and `silent` here?  In my work with partially-typed codebases, I've used the later.  Not sure what would be best for levels 3 and 4.
[Link for context](https://mypy.readthedocs.io/en/stable/running_mypy.html#following-imports)

Reply:
I think you're right, it's better to change it to silent.